app/views/groups/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<nav class="navbar navbar-dark main-bg ">
+  <div class="container">
+      <%= link_to '←', user_path(current_user.id) , class:'text-light text-decoration-none fs-2' %>
+      <a class="navbar-brand" href="#">Spending</a>
+      <%= link_to 'Next',{}, class: 'text-light fs-5 text-decoration-none'%>
+  </div>
+</nav>
+<div class = 'container pt-4 '>
+  <% @groups.each do |group| %> 
+    <% if group.id == params[:id].to_i %>
+      <div class = 'd-flex flex-row justify-content-between w-70'>
+          <div> <%= image_tag group.icon, class:'img' %> </div>
+          <p> <strong><%= group.name %></strong> total spending: <strong>$<%= Group.total_spending(params[:id]) %> </strong></p>
+      <%end%> 
+    <%end%>
+  </div>
+   <div class = 'container py-3'>
+      <% @category_groups.each do |cat_group| %>
+        <% if cat_group.group_id == params[:id].to_i%>
+          <% categories = Category.where(id: cat_group.category_id)%>
+          <% categories.each do |category|%>
+            <div class = 'card border-white'>
+              <div class = 'card-body d-flex flex-row justify-content-between'>
+                <div class = 'd-flex flex-column'>
```

Comment:
### Optional
- [ ] It wold be better if we display the date or created_at in the browser UI as seen in the screenshot below

![Screenshot from 2022-11-19 02-31-37](https://user-images.githubusercontent.com/46388113/202830355-a61fe5e5-a0fd-40e5-a3bc-46f3b0a768bc.png)
![Screenshot from 2022-11-19 02-31-50](https://user-images.githubusercontent.com/46388113/202830356-a36b20c1-c7bc-41cc-afc9-b10f0db083b7.png)

Reply:
I show this date for each expense, but I will continue working on that after the project approval as an enhancement. Thanks for the comments.